.github/workflows/conda_publish.yml

```
@@ -1,50 +1,11 @@
-name: Anaconda Publish
+name: Verify Bioconda Publish
 
 on:
-  release:
-    types: [published]
+  schedule:
+    -cron: '23 23 * * 0,4'
```

It should be run when after matchms is updated on bioconda. So least worse is to use [manual trigger](https://docs.github.com/en/actions/managing-workflow-runs/manually-running-a-workflow).
```suggestion
  workflow_dispatch:
  schedule:
    -cron: '23 23 * * 0,4'
```

Doing it `At 23:23 on Sunday and Thursday.` seems a bit much.